client/src/views/Game.vue

```diff
@@ -73,6 +74,18 @@
               <p>
                 <v-switch v-model="pane_lock" :label="`Unlock Players Pane`"></v-switch>
               </p>
+
+              <!-- Need Help? button -->
+              <v-btn @click.native="helpMenu = !helpMenu" >Need Help?</v-btn>
+              <v-card v-show="helpMenu" class="mt-5 pa-2" outlined tile >                  
+                <router-link to="/help#rules">Rules</router-link> |
+                <router-link to="/help#tutorials">Tutorials</router-link> |
```

Comment:
I believe there is a way to make these look similar to a button. 
`<router-link to="/foo" tag="button">foo</router-link>`
Just a thought though.

Reply:
if someone wants to they can but it's fine how it is, I didn't do that cuz they might be to big